docs/DeveloperGuide.md

```diff
@@ -308,6 +319,27 @@ As seen in the diagram above, after the `LogicManager` receives the `ProfileComm
 6. Finally, `LogicManager` calls `Storage#saveAddressBook(addressBook)` to update the new user profile in the storage and returns
 the `CommandResult` to be displayed to the user.
 
+### View feature
+This feature allows user to view patient's details as PatientTable does not show complete details of the patients.
+
+This feature can be accessed using the `view` command, which currently only support the viewing of patient which follows
+the following format `view t/patient PATIENT_ID` where `PATIENT_ID` is a valid id of a patient.
+
+Given below is the sequence diagram when a user provides an example of a valid `view` command
+(`view t/patient P001`) to see the complete details of the patient.
+
+![ViewPatientCommand](images/ViewSequenceDiagram.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `ViewPatientParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+</div>
+
+As seen in the diagram above, when `view` command is made by the user, following steps happened inside the logic:
+1. When `Logic` is called upon to execute a command, it uses the `AddressBookParser` class to parse the user command.
```

Comment:
All of these actually are pretty redundant, given you have sequence diagram. You can use activity diagram here too btw